src/peregrine/reference/item/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Item
+---
+
+
+The `Item` component takes a data object and renders it using a [render prop] or wraps it in an HTML element. 
+
+This component is used as a direct child of the [`Items`] fragment.
+
+## Props
+
+| Name         | Required                                      | Default | Description                                                    |
+| ------------ | :-------------------------------------------: | ------- | -------------------------------------------------------------- |
+| `classes`    |                                               | `{}`    | A classname hash                                               |
+| `hasFocus`   |                                               | `false` | Set to `true` to get browser focus on the item                 |
```

Comment:
`hasFocus` is non-normative here. Rather than _causing_ the browser to focus the item's underlying element, it merely reflects whether the item is in a focused state or not (insofar as the selection model is concerned).